packages/terra-avatar/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
```

Comment:
This should be a MVB, right? Removing a variant altogether is a breaking change.

Reply:
yes this will be MVB..